File: Java/poo/employeeSalary/library/worker.java
package poo.employeeSalary.library;

public abstract class worker{
    private double salary;
    private int riskLevel;

    public worker(double salary, int riskLevel){
        this.salary = salary;
        this.riskLevel = riskLevel;
    }
    public double getSalary(){
        return this.salary;
    } 
    public int getRiskLevel(){
        return this.riskLevel;
    }
    public abstract double calculateTotalSalary();
}

File: Php/poo/employeeSalary/library/worker.php
<?php

    abstract class worker{

        private float $salary;
        private int $riskLevel;
        
        public function __construct(float $salary, int $riskLevel) {
            $this->salary = $salary;
            $this->riskLevel = $riskLevel;
        }
        public function getSalary(): float{
            return $this->salary;
        }
        public function getRiskLevel(): int{
            return $this->riskLevel;
        }

        abstract public function calculateTotalSalary(): float;
    }
